### DISCUSSION TITLE
Multiple spotlights

### DISCUSSION THREAD

**Discussion by DenyTkachenko** [gilbarbara/react-joyride#894] [Q&A] [answered]
Good afternoon, I really like your library. I would like to be able to highlight multiple elements on the page. I know that there have already been discussions about this, I even saw PR, but as I understand it, for some reason they were not poured. Can you advise me. Is there such a functionality, will it be in the future, or how can I implement it myself?
https://github.com/gilbarbara/react-joyride/issues/389
https://github.com/gilbarbara/react-joyride/pull/572
https://github.com/gilbarbara/react-joyride/pull/730

**Comment by gilbarbara** [comment, marked as answer]
Hey @DenyTkachenko 

I don't plan to add multiple spotlights. You can fork the repo and check the code in the PR you linked
